presto-parser/src/main/java/com/facebook/presto/sql/tree/Identifier.java

```diff
@@ -87,12 +87,12 @@ public boolean equals(Object o)
         }
 
         Identifier that = (Identifier) o;
-        return Objects.equals(value, that.value);
```

Comment:
I agree this should be the case, but i think it isn't yet. `com.facebook.presto.sql.analyzer.AggregationAnalyzer.Visitor#process` uses expression equality. `StatementAnalyzer` does something like `outputExpressions.containsAll(orderByExpressions)` (i didn't search further..)